packages/check-html-links/src/checkLinks.js

```diff
@@ -0,0 +1,59 @@
+import fetch from 'node-fetch';
+
```

Comment:
very straightforward implementation and no dependencis - I like 💪

Reply:
Thanks!